### PR TITLE
[doc][ybm] VictoriaMetrics

### PR DESCRIPTION
**Pull request by ddhodge**
VictoriaMetrics Integration
CLOUDGA-23073

@netlify /preview/yugabyte-cloud/cloud-monitor/managed-integrations/

### CODE REVIEW

File: docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md [hunk 1]
```diff
@@ -127,6 +128,36 @@ To create an export configuration, do the following:
...
+VictoriaMetrics integration is {{<badge/tp>}} and only available for clusters deployed on AWS.
+
+The VictoriaMetrics integration requires the following:
+
+- VictoriaMetrics instance
+  - deployed in a VPC on AWS
+  - publically-accessible endpoint URL that resolves to the private IP of the VictoriaMetrics instance; the DNS for the endpoint must be in a public hosted zone in AWS.
+  - VPC hosting the VictoriaMetrics instance has the following Inbound Security Group rules:
+    - Allow HTTP inbound traffic on port 80 for Prometheus endpoint URL (HTTP)
+    - Allow HTTPS inbound traffic on port 443 for Prometheus endpoint URL (HTTPS)
```
Comment (bansal01yash) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:146]:
```suggestion
    - Allow HTTP inbound traffic on port 80 for VictoriaMetrics endpoint URL (HTTP)
    - Allow HTTPS inbound traffic on port 443 for VictoriaMetrics endpoint URL (HTTPS)
```

File: docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md [hunk 2]
```diff
@@ -127,6 +128,36 @@ To create an export configuration, do the following:
...
+  {{% tab header="VictoriaMetrics" lang="victoria" %}}
+
+VictoriaMetrics integration is {{<badge/tp>}} and only available for clusters deployed on AWS.
+
+The VictoriaMetrics integration requires the following:
+
+- VictoriaMetrics instance
+  - deployed in a VPC on AWS
+  - publically-accessible endpoint URL that resolves to the private IP of the VictoriaMetrics instance; the DNS for the endpoint must be in a public hosted zone in AWS.
+  - VPC hosting the VictoriaMetrics instance has the following Inbound Security Group rules:
```
Comment (bansal01yash) [inline docs/content/preview/yugabyte-cloud/cloud-monitor/managed-integrations.md:144]: Can we add another point here(or somewhere else) specifying that url entered must be of the form specified by VictoriaMetrics docs here and here https://docs.victoriametrics.com/single-server-victoriametrics/#sending-data-via-opentelemetry
https://docs.victoriametrics.com/guides/getting-started-with-opentelemetry/

**Review by aishwarya24 — APPROVED** [review]
LGTM

**Review by bansal01yash — APPROVED** [review]
LGTM, we can set the date on release notes once feature is live and merge

**Pull request merged**